receiver/riakreceiver/metadata.yaml

```diff
@@ -0,0 +1,67 @@
+name: riakreceiver
+
+attributes:
+  riak.node.name:
+    description: The name of the Riak node.
+  operation:
+    value: state
+    description: The operation type.
+    enum:
+      - put
+      - get
+      - read
+      - write
+      - delete
```

Comment:
From what I can tell, these are values from at least two separate attributes.

`get` and `put` are used in the context of requests, and the rest are used in the context of index operations.

We need to split them up appropriately.